chef/cookbooks/horizon/recipes/server.rb

```diff
@@ -165,10 +167,13 @@
 unless monasca_ui_pkgname.nil?
   monasca_servers = search(:node, "roles:monasca-server") || []
   unless monasca_servers.empty?
+    include_recipe "#{@cookbook_name}::monasca_ui"
     package monasca_ui_pkgname do
       action :install
       notifies :reload, resources(service: "apache2")
     end
+    grafana_available = true
```

Comment:
Do you actually need `grafana_available` variable?
You just need checking if `grafana_url` is not empty to include into template.